src/vast_tracker.js

```diff
@@ -459,7 +459,7 @@ export class VASTTracker extends EventEmitter {
       variables['CONTENTPLAYHEAD'] = this.progressFormatted();
     }
 
-    util.track(URLTemplates, variables, options);
+    util.track(util.extractURLsFromTemplates(URLTemplates), variables, options);
```

Comment:
maybe we could move this inside `util.track` to be safe, in case `util.track` will be called elsewhere

Reply:
Done